### ISSUE TITLE
Update M_loss as a cohort effect

### ISSUE THREAD

**Issue by quang-huynh** [Blue-Matter/salmonMSE#15]
Typically, the fitness loss is on the survival term

SAR_loss = SAR_base * fitness^loss

Currently modeling ocean M as

M_loss = M/fitness^loss
